syncd/RedisClient.cpp

```diff
@@ -717,6 +743,36 @@ void RedisClient::insertVidAndRid(
     m_dbAsic->hset(RIDTOVID, strRid, strVid);
 }
 
+void RedisClient::insertVidAndRid(
+        _In_ sai_object_id_t* vids,
+        _In_ sai_object_id_t* rids,
+        _In_ size_t count)
```

Comment:
name is for single vid and rid, change to vid/rids, and also use std::vector maybe ?

Reply:
Done

Reply:
(ptr + size) - more generic, static array, dynamic array, ring buffers, etc.
Ideally, std::span - but it's in c++20